### PR TITLE
Rename MetaData to Metadata

### PR DESCRIPTION
**Pull request by fractalwrench**
## Goal

Renames `MetaData` to `Metadata` to gain consistency with the notifier spec. This changeset also changes any local variable usage of `metaData`.

### CODE REVIEW

File: features/support/env.rb [hunk 1]
```diff
@@ -6,15 +6,15 @@
 # Install latest versions of bugsnag-android
   run_required_commands([
     [
-      "./gradlew", "assembleRelease", "publishToMavenLocal", "-x", "lintVitalRelease"
```
Comment (fractalwrench) [inline features/support/env.rb]: This task is no longer in the project after the example app restructure, so needs to be removed to make mazerunner run.

Comment (bengourley) [reply features/support/env.rb]: Should it have been removed in the the example app restructure PR then?

Comment (fractalwrench) [reply features/support/env.rb]: Ideally yes - this is only needed to run the local version of mazerunner, which is why it was originally missed.

**Comment by fractalwrench** [comment]
Thanks - I've updated the additional cases to use the new naming convention for metadata.

**Comment by fractalwrench** [comment]
Spotted a few more cases myself after doing a global search/replace. I believe that's the last of them.

**Pull request merged**